odoo/custom/src/private/nxpo_procurement_type/models/purchase_order.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 ProThai Technology Co.,Ltd. (http://prothaitechnology.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    def _default_procurement_type(self):
+        return self.env["procurement.type"].search(
+            [],
+            limit=1,
+        )
+
+    procurement_type = fields.Many2one(
+        comodel_name="procurement.type",
+        string="Procurement Type",
+        ondelete="restrict",
+        default=lambda self: self._default_procurement_type(),
```

Comment:
let's shorten code where possible.
```suggestion
        default=lambda self: self.env["procurement.type"].search([], limit=1),
```

Reply:
ข้อนี้แก้ไขไม่ให้ default แล้วค่ะ กลัวว่า user จะไม่เลือกเอง และทำรายการผิดประเภท

Reply:
เอาออกไปเลยหรือครับ (พี่ comment ให้แก้ แต่ไม่ได้เอาออกนะ) 
ยังไงก็ได้นะ พี่ไม่แน่ใจว่าอยากทำแบบไหน